src/hardware/gus.cpp

```diff
@@ -633,6 +646,13 @@ void Gus::AudioCallback(const uint16_t requested_frames)
 		++v;
 	}
 
+	// Pre-scale the stream by the user-defined mixer level
+	auto val = accumulator.begin();
+	while (val < accumulator.end()) {
```

Comment:
Nitpick: how about `val != accumulator.end()`, so the code works even if the container gets changed for whatever reason?

Nitpick: I missed this during the previous review… An array called "accumulator" messes up my functional-programming-oriented brain. Accumulators are usually values accumulating the result of subsequent operations (e.g. sum or product values are stored in accumulators). But it's not really topic of this PR.

Reply:
Yup.. this is an array of accumulators across a time sequence. Each point in this array is the accumulation of the current active voice amplitudes, at the same corresponding point in time.

Reply:
>  how about val != accumulator.end()

This specific code snip is gone, but was able to make a couple similar edits in other iterator loops!